### ISSUE TITLE
Map Icons

### ISSUE THREAD

**Issue by RedPandaOverlord** [RandyKnapp/Auga#48]
I'm trying to put the icons on the map, like double clicking it and everything, but they just disappear and I can't get them to show up anymore

**Comment by Karmacopper** [comment]
Are you using Explore together? I had this same issue, and it turned it to be that.

**Comment by RedPandaOverlord** [comment]
Nope, I have very little mods since the others aren't updated for H&H

**Comment by Lucas407** [comment]
Do you use Valheim Plus? If the map section is enabled icons don't seem to work with this mod

**Comment by RedPandaOverlord** [comment]
Ooooh, alright then! I'll test it out! Thank you!

**Comment by RedPandaOverlord** [comment]
UPDATE : so for valheim plus, just disable the map in valheim plus cause it seems to cause conflict.

[ Map ]

; Change false to true to enable this section. https://valheim.plus/documentation/list#Map
enabled = false

**Comment by SiIvervale** [comment]
Is it possible to get a fix for this? Would love to be able to keep ship markers and player locations!

**Comment by Funky18229** [comment]
this error is a conflict with auga and epicloot adventures, it is putting
 out a mini map null ref which is causing many other issues. PLEASE randy fix this?
